mono/utils/mono-threads-coop.c

```diff
@@ -198,7 +198,7 @@ copy_stack_data_internal (MonoThreadInfo *info, MonoStackData *stackdata_begin,
 	state->gc_stackdata_size = stackdata_size;
 }
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && !defined(TARGET_ARM64)
```

Comment:
So this block has no implementation under ARM64 and will lead to linker error (for `copy_stack_data_internal_win32_wrapper`) if used? I think that's fine, just confirming understanding.

Reply:
yes